packages/block-editor/src/components/writing-flow/use-tab-nav.js

```diff
@@ -56,18 +53,10 @@ export default function useTabNav() {
 		}
 	}
 
-	const before = (
```

Comment:
Since block toolbar is no longer receiving focus via Shift+Tab, remove the before.